src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/coverage/CoverageRenderer.kt

```diff
@@ -50,7 +66,34 @@ class CoverageRenderer(private val color: Color, private val lineNumber: Int, pr
             )
         }
 
-        val panel = prePanel.addVerticalGap(10).panel
+        val state = ApplicationManager.getApplication().getService(TestGenieSettingsService::class.java).state
+        if (coveredMutation.isNotEmpty() && state.criterionWeakMutation) {
+            prePanel.addComponent(JBLabel(" Covered mutants:"), 10)
+            for (mutantName in coveredMutation) {
+                prePanel.addComponent(
+                    ActionLink(mutantName.substringBefore('(')) {
```

Comment:
Why do you take a substring?

Reply:
We take the substring, because EvoSuite always provides us mutation information in the form of `Mutation (id) - line number` and we do not need to provide to the user the id of the mutation or the line number, because the former is EvoSuite internal and the latter is seen in the gutter.